src/components/stats/Stats.liguidity.stories.tsx

```diff
@@ -0,0 +1,171 @@
+
+    const data = [
+      {
+        id: 'current range',
+        data: [
+          { x: '1AM', y: 20 },
+          { x: '4AM', y: 15 },
+          { x: '7AM', y: 10 },
+          { x: '10AM', y: 9 },
+          { x: '1PM', y: 5 },
+          { x: '4PM', y: 3 },
+          { x: '7PM', y: 10 },
+          { x: '10PM', y: 5 }
+        ]
+      }
+    ]
+
+    const Theme = {
+      axis: {
+        tickColor: 'transparent',
+        ticks: { line: { stroke: colors.invariant.component }, text: { fill: colors.white.main } }
+      }
+    }
+
+    const classes = styles()
+
+    return (
+      <Grid className={classes.container}>
+        <Box className={classes.liquidityContainer}>
+          <Typography className={classes.liquidityHeader}>Liquidity 24H</Typography>
+          <div className={classes.volumePercentHeader}>
+            <Typography className={classes.volumeLiquidityHeader}>
+              ${formatNumbers()(liquidityVolume.toString())}
+              {showPrefix(liquidityVolume)}
+            </Typography>
+            <Box className={classes.volumeStatusContainer}>
+              <Box
+                className={classNames(
+                  classes.volumeStatusColor,
+                  liquidityPercent < 0 ? classes.backgroundVolumeLow : classes.backgroundVolumeUp
+                )}>
+                <Typography
+                  component='p'
+                  className={classNames(
+                    classes.volumeStatusHeader,
+                    liquidityPercent < 0 ? classes.volumeLow : classes.volumeUp
+                  )}>
+                  {liquidityPercent < 0
+                    ? `-${liquidityPercent.toString().split('-')[1]}`
+                    : `+ ${liquidityPercent}`}
+                  %
+                </Typography>
+              </Box>
+            </Box>
+          </div>
+        </Box>
+        <div className={classes.barContainer}>
+          <ResponsiveLine
+            data={data}
+            margin={{ top: 70, bottom: 30, left: 10, right: 20 }}
+            axisBottom={{
+              tickSize: 0,
+              tickPadding: 0,
+              tickRotation: 0,
+              tickValues: 5
+            }}
+            curve={'monotoneX'}
+            enableGridX={false}
+            enableGridY={false}
+            enablePoints={false}
+            enableArea={true}
+            isInteractive={false}
+            animate={false}
+            colors={colors.invariant.green}
+            theme={Theme}
+            lineWidth={1}
+            defs={[
+              linearGradientDef('gradient', [
+                { offset: 0, color: 'inherit' },
+                { offset: 50, color: 'inherit' },
+                { offset: 100, color: 'inherit', opacity: 0 }
+              ])
+            ]}
+            fill={[{ match: '*', id: 'gradient' }]}
+          />
+        </div>
+      </Grid>
+    )
+  })
+
+const styles = makeStyles(() => ({
```

Comment:
try to use "useStyles" name instead of "styles". Try to keep the nomenclature

Reply:
done